plugin/src/main/java/com/denizenscript/denizen/objects/LocationTag.java

```diff
@@ -1249,6 +1249,23 @@ public static void register() {
             }
         });
 
+        // <--[tag]
+        // @attribute <LocationTag.waxed>
+        // @returns ElementTag(Boolean)
+        // @mechanism LocationTag.waxed
+        // @group world
+        // @description
+        // Returns whether the location is a Sign block that is waxed.
+        // -->
+        tagProcessor.registerTag(ElementTag.class, "waxed", (attribute, object) -> {
+            BlockState state = object.getBlockStateForTag(attribute);
+            if (!(state instanceof Sign)) {
```

Comment:
Should use `instanceof` pattern matching, I.e. `instanceof Sign sign` & use `sign`